inspirehep/modules/orcid/models.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Models for ORCID integration."""
+
+from __future__ import absolute_import, division, print_function
+
+from invenio_db import db
+
+
+class InspireOrcidPutCodes(db.Model):
+    __tablename__ = 'inspire_orcid_put_codes'
+
+    recid = db.Column(db.Integer, primary_key=True, index=True)
+    put_code = db.Column(db.Integer, nullable=False, index=True)
```

Comment:
@szymonlopaciuk weren't you saying that put codes are different on different ORCID profiles, even if we have the same record for it? this would mean that you need to associate a put code not to an INSPIRE recid, but to a (recid, orcid) pair.

Reply:
I was, thanks :+1: